src/libOpenImageIO/color_ocio.cpp

```diff
@@ -2348,7 +2348,7 @@ colorconvert_impl(ImageBuf& R, const ImageBuf& A,
         unpremult = false;
     // clang-format off
     parallel_image(
-        roi, parallel_options(nthreads),
+        roi, paropt(nthreads),
```

Comment:
Looks like there's a, final, reference to `parallel_options` on line 2437 of this file still?  That said, the regression in class name to something so terse is a bit unfortunate :-/

Reply:
Good catch, thanks. Fix pushed.

I hear what you're saying about the name, but the parallel_options -> paropt is something that happened years ago, I'm just finally clearing away the old structure definition. The good thing is that the paropt parameters are almost always either defaulted, or just pass the thread count (paropt has a ctr for just that), so almost nobody external ever actually types "paropt."

Reply:
I see.  This recent commit also removes the last usage of the `parallel_options` class altogether. Should it be removed too?

Reply:
Unfortunately, it's referenced in the public API of an OSL header. I need to do this incredibly minor API transition in OSL before I can remove it here, or it just creates a different headache.